torch/_meta_registrations.py

```diff
@@ -88,6 +88,26 @@ def meta_take(self, index, *, out=None):
     return result
 
 
+@register_meta([aten.cummax.default, aten.cummax.out])
+@out_wrapper("values", "indices")
+def cummax(self, dim):
+    values, indices = torch.empty_like(self), torch.empty_like(self, dtype=torch.int64)
+    if self.numel() != 0 and self.ndim != 0:
+        # Checks that dim is within bounds
+        maybe_wrap_dim(dim, self.ndim)
+    return values, indices
```

Comment:
numel/ndim checks are based on `cummax_out`

Reply:
You can maybe just use the same function for both cummax and cummin?